python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -1002,13 +1096,63 @@ async def do_stopTracking(self, data):
         self.assert_enabled()
         await self.send_command(commands.BothAxesStop(), do_lock=False)
 
+    def handle_available_settings(self, reply):
+        """Handle a `ReplyId.AVAILABLE_SETTINGS` reply."""
```

Comment:
That's a smart way of compacting the method documentation but doesn't this violate our coding standards? I see this also a few lines down.

Reply:
If you are referring to not having the trailing `"""` on its own line: black insists on doing it this way.

Note that this can be a nuisance if a brief description is just long enough that putting it all on one line goes past 80 characters because that makes flake8 unhappy. One ends up having to modify the doc string.

Reply:
I was referring to the fact that the documentation doesn't have a `Parameters` setting where the `reply` parameter is explained.

Reply:
Frankly: no. Not with the current documentation. I was merely wondering about our coding standards. I am good with the current way of documenting this and some other methods.